WooCommerce/WooCommerceTests/ViewRelated/Orders/OrderListCellViewModelTests.swift

```diff
@@ -70,7 +70,7 @@ final class OrderListCellViewModelTests: XCTestCase {
         let viewModel = OrderListCellViewModel(order: order, currencySettings: ServiceLocator.currencySettings)
 
         // Then
-        XCTAssertEqual(viewModel.salesChannel, OrderListCellViewModel.Localization.salesChannelPOSText)
+        XCTAssertEqual(viewModel.salesChannel, "POS")
```

Comment:
I had to revert the localization here as now the localized string sits in Networking 🤔